torch/onnx/symbolic_opset9.py

```diff
@@ -681,7 +690,9 @@ def leaky_relu(g, input, negative_slope, inplace=False):
 
 @parse_args('v', 'i')
 def glu(g, input, dim):
-    assert input.type().sizes()[dim] % 2 == 0
+    dim_size = sym_help._get_tensor_dim_size(input, dim)
```

Comment:
Same question for glu: do we have a unit test?

Reply:
added for glu